src/assets/css/librarybrowser.scss

```diff
@@ -208,15 +246,16 @@
     align-items: center;
     text-decoration: none;
     color: inherit;
-    padding: 0.9em 0 0.9em 2.4em !important;
+    padding-block: 0.9em !important;
+    padding-inline: 2.4em 0 !important;
     flex-grow: 1;
     font-weight: 400 !important;
     margin: 0 !important;
     border-radius: 0 !important;
 }
 
 .navMenuOptionIcon {
-    margin-right: 1.2em;
+    margin-inline-end: 1.2em;
```

Comment:
Unfortunately, I believe some of these newer CSS properties will cause issues on the old browsers shipped on TVs (webOS and Tizen specifically). Someone will need to test those.

Reply:
Indeed. I'll replace them with appropriate [dir="rtl"] selectors.

Reply:
Fixed